.buildkite/scripts/run_e2e_tests.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VERSION_URL="https://storage.googleapis.com/artifacts-api/releases/current"
```

Comment:
@dliappis 🎉 initiative!

Reply:
I have requested a review, I would appreciate if you can review at least the pipeline setup and shell scripts. Python scripts are big but based on KISS since alot logics. As much possible, tried to add comments for edge cases, others read source as going.
Thanks.